_unused_topics/cluster-logging-exported-fields-kubernetes.2021-06-04.adoc

```diff
@@ -0,0 +1,83 @@
+[id="cluster-logging-exported-fields-kubernetes_{context}"]
+= Kubernetes
+
+The following fields can be present in the namespace for kubernetes-specific metadata.
+
+== kubernetes.pod_name
+
+The name of the pod
+
+[horizontal]
+Data type:: keyword
+
+
+== kubernetes.pod_id
+
+Kubernetes ID of the pod.
```

Comment:
This one isn't a sentence, but has a period. The previous one doesn't have a period. Further down, the descriptions are complete sentences and have a period, correctly.

modules/cluster-logging-exported-fields-kubernetes.adoc more consitenly uses periods at the end of sentences and omits them when they're not sentences.

Reply:
Thanks, updated.

Reply:
Unused file.